README.md

```diff
@@ -287,7 +287,7 @@ justify the resolution on a global scale.
 |-------------|--------|-----------------------------------|
 | YAML / JSON | Global | `$ORT_CONFIG_DIR/resolutions.yml` |
 
-#### [Repository configuration file](./docs/config-file-ort-yml.md)
+#### [Repository configuration file](./website/docs/configuration/ort-yml.md)
```

Comment:
Looks like all title links in this section starting with "Copyright garbage file" are broken. Mind fixing them as well?

Reply:
I've fixed some https://github.com/oss-review-toolkit/ort/pull/7700.

Reply:
Actually @nnobelis [had already fixed the other links](https://github.com/oss-review-toolkit/ort/compare/64da439b0191e50a44caee39a873a844bc70abcb..0061f6b0f730ccc0026027cc2efaa6cb4ea89482) in this PR. Let's take this PR from him and rebase yours on top.